### PR TITLE
docs(bindable): minor changes

### PR DESCRIPTION
**Pull request by Sayan751**
# Pull Request

### CODE REVIEW

File: docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md [hunk 1]
```diff
@@ -22,16 +22,47 @@ export class Loader {
...
 <loader loading.bind="loadingVal"></loader>
 ```
 
+You can also bind values without the `.bind` part, as can be seen the following example.
+
+```html
+<loader loading="true"></loader>
+```
+
+However, there is a subtle difference. In this case, Aurelia consider the attribute value as a string. Thus, instead of a boolean `true` value the string `'true'` gets bound to the `loading` property. This might cause some unexpected issues. However, you can apply coercion using a [bindable setter](bindable-setter.md) or [defining the bindable datatype explicitly](bindable-coercion.md).
```
Comment (CollinHerber) [inline docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md:43]: > Aurelia consider the attribute value as a string.
-> Aurelia `considers` the attribute value as a string.

File: docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md [hunk 2]
```diff
@@ -22,16 +22,47 @@ export class Loader {
...
+
+## Attribute naming
+
+As can be seen in the examples above, the `@bindable`s defined inside a custom element can be used a HTML attribute while using the custom element. For example, the `@bindable loading` property is used as the attribute `loading.bind` in the view.
```
Comment (CollinHerber) [inline docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md:47]: > can be used a HTML attribute while using the custom element  

-> can be used as an HTML attribute while using the custom element

**Review by CollinHerber — COMMENTED** [review]
Just some minor English adjustments

File: docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md (hunk 1 above)
Comment (Sayan751) [reply docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md]: Thanks for the review!

File: docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md (hunk 2 above)
Comment (Sayan751) [reply docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md]: Thanks for the review!

File: docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md [hunk 3]
```diff
@@ -22,16 +22,47 @@ export class Loader {
 
 This will allow our component to be passed in values. Our specified bindable property here is called `loading` and can be used like this:
 
-```text
-<loader loading="true"></loader>
+```html
```
Comment (bigopon) [inline docs/user-docs/getting-to-know-aurelia/components/bindable-properties/the-basics.md:25]: `html` will be changed to `markup` by gitbook, it's quite an annoying one, I;m not sure how we can handle it to make it more natural to markdown

**Review by bigopon — APPROVED** [review]
Nice doc work 👍

**Pull request merged**